client/app/components/query.jsx

```diff
@@ -1,14 +1,41 @@
 import React from 'react';
 
-const Query = () => (
-  <div>
-    Origin: <input type="text" />
-    Destination: <input type="text" />
-    Time of day: <select>
-      <option>Day</option>
-      <option>Night</option>
-    </select>
-  </div>
-);
+class Query extends React.Component {
```

Comment:
Can the Query class be a functional stateless component and just call the set origanddest function in app?